app/components/buttons/icon-small-button.jsx

```
@@ -6,17 +6,18 @@
  * @param {string} [props.className] - Optional class name for the button
  * @param {Function} [props.onClick] - Optional click handler for the button
  * @param {React.ReactNode} props.children - Child elements to be rendered inside the button
- * @returns {JSX.Element} The rendered TextButton component
+ * @param {bool} [props.disabled] - Optional flag to disable the button
+ *  @returns {JSX.Element} The rendered TextButton component
  */
 export default function IconSmallButton(props) {
   return(
-      <div className="text-btn" role='button' onClick={props.onClick}>
+      <button className="icon-btn disabled:text-[--md-sys-color-outline-variant]" role='button' onClick={props.onClick} disabled={props.disabled}>
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<button>
<input type="button">


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)